oe_content.post_update.php

```diff
@@ -129,3 +130,25 @@ function oe_content_post_update_00004(): void {
     }
   }
 }
+
+/**
+ * Ensure truncate maxlength setting is applied for textfields.
+ */
+function oe_content_post_update_00005(): void {
+  $displays = EntityFormDisplay::loadMultiple();
+  foreach ($displays as $display) {
+    $changed = FALSE;
+    foreach ($display->getComponents() as $field_name => $component) {
+      if ($component['type'] !== 'string_textfield' && empty($component['third_party_settings']['maxlength'])) {
```

Comment:
I suggest we update only the entity form displays of node entities.
If we update the `maxlength_js_enforce` value we should do this for `text_textarea` as well.
Shouldn't be an "or" condition instead of "and"? Since there might be text fields without maxlength set.

We need to make sure `type` is set.
<img width="441" alt="warning" src="https://user-images.githubusercontent.com/22908988/192274449-a573cd10-0269-478e-a853-340b9216b392.png">